packages/components/src/tooltip/index.tsx

```diff
@@ -66,7 +66,7 @@ function Tooltip( props: TooltipProps ) {
 
 	const tooltipStore = Ariakit.useTooltipStore( {
 		placement: computedPlacement,
-		timeout: delay,
+		showTimeout: delay,
```

Comment:
Looking at the `delay` prop, it looks like it was always intended to only be applied to the "show" timeout, and not affect the "hide" timeout.